reark/src/main/java/io/reark/reark/data/stores/StoreGetInterface.java

```diff
@@ -0,0 +1,10 @@
+package io.reark.reark.data.stores;
+
+import rx.Observable;
+
+/**
+ * Created by ttuo on 27/06/16.
+ */
+public interface StoreGetInterface<T, U> {
+    Observable<U> getStream(T id);
```

Comment:
Why should get() return null? I'd rather see it completing without returning anything. The user can always use defaultIfEmpty(null) to get the null behaviour.